yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/call/factory/ProfilePathFactoryTest.java

```diff
@@ -0,0 +1,28 @@
+package com.yoti.api.client.spi.remote.call.factory;
+
+import org.junit.Test;
+
+import java.net.URI;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ProfilePathFactoryTest {
+
+    private static final String SOME_APP_ID = "someAppId";
+    private static final String SOME_TOKEN = "someToken";
+
+    ProfilePathFactory testObj = new ProfilePathFactory();
+
+    @Test
+    public void shouldCreate() {
```

Comment:
shouldCreateToken, right?

Reply:
Can do.  This is removed in the next commit anyway.

It would have been shouldCreatePath.